sdk/dns/Azure.ResourceManager.Dns/api/Azure.ResourceManager.Dns.netstandard2.0.cs

```diff
@@ -561,21 +561,21 @@ public partial class DnsZonePatch
         public DnsZonePatch() { }
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
     }
-    public partial class MxRecord
+    public partial class MXRecord
     {
-        public MxRecord() { }
+        public MXRecord() { }
         public string Exchange { get { throw null; } set { } }
         public int? Preference { get { throw null; } set { } }
     }
-    public partial class NsRecord
+    public partial class NSRecord
     {
-        public NsRecord() { }
-        public string Nsdname { get { throw null; } set { } }
+        public NSRecord() { }
+        public string NsdName { get { throw null; } set { } }
     }
     public partial class PtrRecord
     {
         public PtrRecord() { }
-        public string Ptrdname { get { throw null; } set { } }
+        public string PtrdName { get { throw null; } set { } }
     }
     public partial class SoaRecord
```

Comment:
Prepend `Dns` 

Also for other models with simple name, like the `ZoneType`

Reply:
NsdName => DnsNsdName
Ptrdname => DnsPtrdname
ZoneType => DnsZoneType
ZoneListResult => DnsZoneListResult (Considering that all other `Zone` prefixed models are named `DnsZone`)